src/payment/payment-methods/cybersource/service.js

```diff
@@ -19,36 +20,47 @@ export function configureApiService(newConfig, newApiClient) {
   apiClient = newApiClient;
 }
 
-export async function sdnCheck(firstName, lastName, city, country) {
+export async function sdnCheck(basketId, firstName, lastName, city, country) {
   const { data } = await apiClient.post(
     `${config.ECOMMERCE_BASE_URL}/api/v2/sdn/search/`,
     {
       name: `${firstName} ${lastName}`,
       city,
       country,
     },
-  );
+  ).catch((error) => {
+    logAPIErrorResponse(error, {
+      messagePrefix: 'SDN Check Error',
+      paymentMethod: 'Cybersource',
+      paymentErrorType: 'SDN Check',
+      basketId,
+    });
+
+    throw error;
+  });
 
   return data;
 }
 
 export async function checkout(basket, { cardHolderInfo, cardDetails }) {
+  const { basketId } = basket;
+
   const sdnCheckResponse = await sdnCheck(
+    basketId,
     cardHolderInfo.firstName,
     cardHolderInfo.lastName,
     cardHolderInfo.city,
     cardHolderInfo.country,
   );
 
   if (sdnCheckResponse.hits > 0) {
+    /* istanbul ignore next */
     if (config.ENVIRONMENT !== 'test') {
-      /* istanbul ignore next */
       global.location.href = `${config.ECOMMERCE_BASE_URL}/payment/sdn/failure/`;
     }
-    throw new Error('SDN Failure');
+    throw new Error('This card holder did not pass the SDN check.');
```

Comment:
What happens to this and this message?  Is it logged?  I assume not, but if it were, it should only be INFO.

Reply:
This is not logged